app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    elements = command.split()
+    if len(elements) == 3:
+        command, file_in, file_path = elements
+        if command == "mv":
+            path_elements = file_path.split("/")
+            path = path_elements[0]
+            for element in path_elements[1:]:
```

Comment:
Also, try to do this without using loop. There is a function in os lib for creating several dirs.